src/App.tsx

```diff
@@ -1,14 +1,19 @@
+import { useEffect } from "react"
 import sdk from "./sdk"
 
 function App() {
 
+  useEffect(() => {
+    sdk.listen(window, 'http://localhost:3000');
```

Comment:
let's keep sdk listen, inside sdk initalization, no need for duplicates

Reply:
In practice, there will be two listeners. One on the parent page, and one on the iframe. 
In our case, the <App /> component is the parent and the <Login /> is from the iframe. Hence the two separate listeners